<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameRoom</class>
 <widget class="QDialog" name="GameRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0,200);</string>
  </property>
  <widget class="FiveNodesWidget" name="WG_Game" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Icon0">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>480</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Name0">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Name1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>130</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Icon1">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_AImode">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>660</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>AI托管</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LB_MINECOLOR">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>650</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>自己的棋子颜色</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_WHOSETURN">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>340</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>谁的回合</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_ENEMYCOLOR">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>210</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>对方的棋子颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Start">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>670</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Host1">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Host0">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_Count">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>计时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FiveNodesWidget</class>
   <extends>QWidget</extends>
   <header>fivenodeswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
